crates/experimentation_platform/src/api/experiments/helpers.rs

```diff
@@ -37,6 +37,16 @@ pub fn check_variant_types(variants: &Vec<Variant>) -> superposition::Result<()>
     Ok(())
 }
 
+pub fn validate_context(context: &Value) -> superposition::Result<()> {
```

Comment:
This will done on the CAC side, we do compile of the schema provided for default configs and the dimensions, but this is missing while creating context, this can be replicated.
Will do this another PR, can you create an issue for this?